.gitlab/image.yml

```diff
@@ -75,9 +75,21 @@
     - amdci
     - gpu
 
-.use_gko-oneapi:
+.use_gko-oneapi-cpu:
   image: ginkgohub/oneapi:latest
   tags:
     - private_ci
-    - bwuni
+    - intelci-igpu
     - cpu
+
+.use_gko-oneapi-igpu:
+  image: ginkgohub/oneapi:latest
+  tags:
+    - private_ci
+    - intel-igpu
```

Comment:
one is intelci-igpu but another is intel-igpu

Reply:
In the second case any IGPU would work, but for the first one I want to make sure the job is executed on that machine so that I can reevaluate where to put this job if we get another Intel CPU.
For example, if we have a runner on an Intel Cascadelake or another new CPU, this might be a good place to try to run this job on.

Reply:
I see.